src/components/MovieCard/MovieCard.tsx

```diff
@@ -3,14 +3,14 @@ import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
 type Props = {
-  movie: Movie;
+  movie: Movie | null;
```

Comment:
```suggestion
  movie: Movie;
```
`movie` should be required, you don't need to render a `MovieCard` component without it